lib/cosmos/top_level.rb

```diff
@@ -296,7 +296,8 @@ def self.run_process_check_output(command)
     Cosmos.set_working_dir do
       thread = Thread.new do
         output, _ = Open3.capture2e(command)
-        if !output.empty?
+        # TODO: Remove check for "Untested Windows.." when Qtbindings updated
+        if !output.empty? && !output.include?("Untested Windows version 10.0")
```

Comment:
This will suppress output on Windows 10 even if there is more than the one line we don't want to show